src/components/settings/forms/Sector.vue

```diff
@@ -479,6 +494,7 @@ export default {
     padding: 0.75rem 1rem;
```

Comment:
You can use the token variables we have in unnnic. Here are all the tokens
https://www.figma.com/design/YbvOpVPUBxTSEzg4J8pXKu/Weni---unnnic?node-id=4674-21107&m=dev